### PR TITLE
Improvement of the Data column explanation + updating tenants PR (#88)

### PR DESCRIPTION
**Pull request by lukinovec**
Original PR: https://github.com/stancl/tenancy-docs/pull/88/files

### CODE REVIEW

**Comment by stancl** [comment]
Are all the reviews in #88 resolved now?

**Comment by lukinovec** [comment]
> Are all the reviews in #88 resolved now?

Resolved them now @stancl

File: source/docs/v3/tenants.blade.md [hunk 1]
```diff
@@ -60,17 +59,29 @@ After the tenant is created, an event will be fired. This will result in things
 
 ## Custom columns {#custom-columns}
 
-Attributes of the tenant model which don't have their own column will be stored in the `data` JSON column.
+Attributes of the tenant model which don't have their column will be stored in the `data` JSON column. You can set these attributes as you'd normally set model attributes:
```
Comment (stancl) [inline source/docs/v3/tenants.blade.md:62]: - The first sentence doesn't need to be changed, add the `own` back
- updated second sentence
```suggestion
Attributes of the tenant model which don't have their own column will be stored in the `data` JSON column. You can set these attributes like you'd set normal model attributes:
```

File: source/docs/v3/tenants.blade.md [hunk 2]
```diff
@@ -60,17 +59,29 @@ After the tenant is created, an event will be fired. This will result in things
...
+
+```php
+$tenant->update([
+    'attributeThatHasNoColumn' => 'value', // stored in the `data` JSON column
+    'plan' => 'free' // stored in the `plan` column (see below)
```
Comment (stancl) [inline source/docs/v3/tenants.blade.md:67]:
```suggestion
    'plan' => 'free' // stored in the dedicated `plan` column (see below)
```

File: source/docs/v3/tenants.blade.md [hunk 3]
```diff
@@ -60,17 +59,29 @@ After the tenant is created, an event will be fired. This will result in things
...
+```
+or
+```php
+$tenant->customAttribute = 'value'; // stored in the `data` JSON column
+$tenant->plan = 'free'; // stored in the `plan` column (see below)
+$tenant->save();
+```
 
-You may define the custom columns by overriding the `getCustomColumns()` method on your `Tenant` model:
+You may define the custom columns that **won't** be stored in the `data` JSON column by overriding the `getCustomColumns()` method on your `Tenant` model:
```
Comment (stancl) [inline source/docs/v3/tenants.blade.md:77]:
```suggestion
You may define the custom columns (that **won't** be stored in the `data` JSON column) by overriding the `getCustomColumns()` method on your `Tenant` model:
```

**Comment by stancl** [comment]
if the changes look good to you, you can commit the suggestions and merge the PR

**Pull request merged**